support-frontend/assets/helpers/subscriptions.js

```diff
@@ -9,7 +9,7 @@ import {
   Quarterly,
 } from 'helpers/billingPeriods';
 import { trackComponentEvents } from './tracking/ophan';
-import { gaEvent } from './tracking/googleTagManager';
```

Comment:
You should be able to remove the ga event tag from tagmanager once this is merged
<img width="636" alt="Screen Shot 2021-01-25 at 10 13 35" src="https://user-images.githubusercontent.com/181371/105692240-13acf580-5ef6-11eb-8c59-042ef28f9468.png">

Reply:
I'll need to cross-check with the Contributions team, the function they use for all their tracking calls uses Ophan and GA, so they may be using it still, also I need to check whether `gaEvent` is used for clicks alone.

Reply:
Did this ever get removed?

Reply:
No it's still in place within GA, it sits behind the Google Analytics vendor consent check.